Payload/Generator.php

```diff
@@ -51,19 +29,11 @@ public function getExceptionPayload($exception): array
             'body'             => [],
             'framework'        => Kernel::VERSION,
             'server'           => $this->getServerInfo(),
-            'language_version' => phpversion(),
+            'language_version' => PHP_VERSION,
```

Comment:
AFAIK we don't send language versions via the API. I might be wrong but I am pretty sure this does not do anything. We might be able to completely remove it.

Reply:
Author here. I can remove this if you wish. Any other comments on this PR to move things forward?

Reply:
Thanks Chris, I am going to merge this with the base 5.x branch. If you would like you can submit a PR to that branch removing it.